k9mail-library/src/main/java/com/fsck/k9/mail/internet/MimeMessage.java

```diff
@@ -2,19 +2,6 @@
 package com.fsck.k9.mail.internet;
 
 
-import java.io.BufferedWriter;
```

Comment:
Please import our [code style settings](https://github.com/k9mail/k-9/wiki/CodeStyle) or configure your IDE to not reorder imports.